packages/backend-next/package.json

```diff
@@ -25,10 +25,17 @@
     "clean": "backstage-cli package clean"
   },
   "dependencies": {
+    "@backstage/backend-common": "workspace:^",
     "@backstage/backend-defaults": "workspace:^",
+    "@backstage/backend-plugin-api": "workspace:^",
     "@backstage/plugin-app-backend": "workspace:^",
     "@backstage/plugin-catalog-backend": "workspace:^",
     "@backstage/plugin-scaffolder-backend": "workspace:^",
+    "@backstage/plugin-search-backend": "workspace:^",
+    "@backstage/plugin-search-backend-module-catalog": "workspace:^",
+    "@backstage/plugin-search-backend-module-explore": "workspace:^",
+    "@backstage/plugin-search-backend-module-techdocs": "workspace:^",
+    "@backstage/plugin-search-backend-node": "workspace:^",
```

Comment:
Couple of extra deps that aren't needed here right?

Reply:
will take a look!